src/viam/examples/modules/tflite/main.cpp

```diff
@@ -93,7 +93,7 @@ class MLModelServiceTFLite : public vsdk::MLModelService, public vsdk::Stoppable
         }
     }
 
-    void reconfigure(vsdk::Dependencies dependencies, vsdk::ResourceConfig configuration) final
+    void reconfigure(const vsdk::Dependencies& dependencies, const vsdk::ResourceConfig& configuration) final
```

Comment:
👍 sorry I missed this!

Reply:
No worries - I actually forgot about it too, but then I was going to actually use this module for something and realized it was out of date!